auth/src/main/java/org/aerogear/auth/IRole.java

```diff
@@ -2,4 +2,6 @@
 
 public interface IRole {
     String getRoleName();
+    RoleType getRoleType();
+    String getClientID();
```

Comment:
This is just for ClientRole, so should not be in the general interface

Reply:
Can this function be kept here if your comment below is addressed?

Reply:
I think it would be better to remove it, since it is not a property of all the roles